themes/gfsc/layouts/index.html.html

```diff
@@ -24,6 +24,30 @@ <h3>{{$vennData.Params.center}}</h3>
 	 </div>
 {{ end }}
 
+
+<hr class="fancy fancy--no-phone" />
+{{ with $homepage.Resources.GetMatch "themes.md" }}
+  {{ $themes := $.Site.GetPage "/our-work/theme/" }}
+  <div class="homepage__themes">
+    <h2 class="homepage__themes__title">{{ .Title }}</h2>
+    <div class="homepage__themes__content">{{ .Content }}</div>
+    <ul class="windows windows--themes windows--orange">
+      {{ range $themes.RegularPages }}
+        <li class="window">
+          <h3 class="window__header">{{ .Title }}</h3>
+          <p class="window__content">{{ .Description }}</p>
+          <div class="window__footer">
+            <div class="window__footer__buttonbox">
+              <a class="window__footer__button" href="{{ .Permalink }}">Browse</a>
```

Comment:
Can we switch this css to one of the btn classes from `_buttons.sass` because we're getting this weird issue where you can see a bit of the background. It was happening in lots of places and I fixed it with 

```css
  background-size: 210% 100%
```
But because we removed this code it never got cleaned up.

Reply:
Hmm I have changed the class to 'btn' and now the background is no longer peeking through

Reply:
nope i think i made it worse